### DISCUSSION TITLE
Add snapshot_name snapshot_arn to redis module in service catalog

### DISCUSSION THREAD

**Discussion by fsanzdev** [gruntwork-io/knowledge-base#185] [Help] [answered]
Hi, 

It looks like `snapshot_name` and `snapshot_arn` options are supported in the redis module `https://github.com/gruntwork-io/terraform-aws-cache/blob/master/modules/redis/main.tf` but they are not available in the service catalog redis module `https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/data-stores/redis/variables.tf` 

Would it be possible to add them?

Thanks,
Fernando.

**Comment by yorinasub17** [comment, marked as answer]
Yup we can definitely expose this! This is being done in https://github.com/gruntwork-io/terraform-aws-service-catalog/pull/1211.

NOTE: since this is a request for a module enhancement, in the future it would be best to post this as a github issue in the relevant repo (in this case, `terraform-aws-service-catalog`).
